middle_end/flambda2/terms/flambda_primitive.mli

```diff
@@ -101,6 +115,7 @@ module Duplicate_block_kind : sig
           length : Targetint_31_63.t
         }
     | Naked_floats of { length : Targetint_31_63.t }
+    | Mixed
```

Comment:
Either add the tag/shape here or note that the fields are not necessary because they're only used for printing